models/repo_list.go

```diff
@@ -110,11 +110,37 @@ type SearchRepoOptions struct {
 	// maximum: setting.ExplorePagingNum
 	// in: query
 	PageSize int `json:"limit"` // Can be smaller than or equal to setting.ExplorePagingNum
+	// Type of repository to search
+	//
+	// in: query
```

Comment:
Could you please run `make generate-swagger` to regenerate the swagger spec? This might resolve the conflict. If not please resolve the conflicts manually.